app/middleware/getPlaces.js

```diff
@@ -0,0 +1,43 @@
+const log = require('../lib/logger');
+const locate = require('../lib/locate');
+const sortByLocalType = require('../lib/sortByLocalType');
+const renderer = require('./renderer');
+const placeSearches = require('../lib/promCounters').placeSearches;
+const placeDisambiguationViews = require('../lib/promCounters').placeDisambiguationViews;
+
+// to do move to common library
+function renderFindHelpPage(req, res, location, message, errorMessage) {
+  log.info({ locationValidationResponse: { location } }, message);
+  res.locals.errorMessage = errorMessage;
+  renderer.findHelp(req, res);
+}
+
+function logZeroResults(places, location) {
+  if (places.length === 0) {
+    log.warn({ location }, `No results were found for ${location}`);
+  }
+}
+
+function getLocationUrl(place, location) {
+  return `results?location=${location}&latitude=${place.latitude}&longitude=${place.longitude}`;
```

Comment:
Does `location` need URL encoding?

Reply:
this will be removed when combine the routes